service/src/internal/inclusion.rs

```diff
@@ -92,9 +96,14 @@ impl InclusionService {
         debug!("Job worker started");
         while let Some(Some(job)) = job_receiver.recv().await {
             let service = self.clone();
+            let metrics = self.metrics.clone();
+            metrics.jobs_attempted.inc();
             tokio::spawn(async move {
                 debug!("Job worker received {job:?}",);
-                let _ = service.prove(job).await; //Don't return with "?", we run keep looping
+                let _ = service
+                    .prove(job)
+                    .await
+                    .map_err(|e| count_error(&metrics, e)); //Don't return with "?", we run keep looping
```

Comment:
nice! all errors can be caught for the service here (AFAIK)

Reply:
cool